app/controllers/districts_controller.rb

```diff
@@ -50,6 +50,17 @@ def sign_public_key
     render json: json
   end
 
+  def get_ssm_parameter
```

Comment:
Its better to make this a controller `SsmParametersController` and then just use the standard rails actions, then in the route just have 

```
resources :ssm_parameter, only: [:index, :create, :destroy]
```

Have a look at the `HeritageController` on how to do this. This means that your action that you are implementing right now is just `show` and you do not have to explicitly declare it in the routes.rb